apps.json

```diff
@@ -1157,6 +1157,13 @@
         "state": "notworking",
         "url": "https://github.com/mitra42/internetarchive_ynh"
     },
+    "invoiceninja": {
+        "branch": "master",
+        "category": "productivity_and_management",
+        "revision": "HEAD",
```

Comment:
If your package points to HEAD, please create a branch `testing`, code into that branch, and when a new version is ready, merge `testing` into `master`.

Reply:
Sure thing. From the readme it didn't read like a must-have.

Reply:
Fixed.